client/src/components/MainPage/index.jsx

```diff
@@ -1,15 +1,53 @@
 import React, { Fragment, Component } from 'react';
 import ProgressBar from '../common/ProgressBar';
 import ProfileCard from '../common/ProfileCard';
+import SpecializationCard from './SpecializationCard';
+import SearchBar from '../common/SearchBar';
+import cardiacImg from '../../assets/cardiac.png';
+import orthoImg from '../../assets/orthopedics.png';
+import generalImg from '../../assets/general.png';
+import psychImg from '../../assets/psychiatry.png';
 import './style.css';
 
 class MainPage extends Component {
-  state = {};
+  state = {
+    values: [
+      {
+        id: 1,
+        imagePath: cardiacImg,
+        section: 'Cardiac Section',
+        questions: '50 Question',
+      },
+      {
+        id: 2,
+        imagePath: orthoImg,
+        section: 'Orthopedics Section',
+        questions: '50 Question',
+      },
+      {
+        id: 3,
+        imagePath: generalImg,
+        section: 'General Section',
+        questions: '50 Question',
+      },
+      {
+        id: 4,
+        imagePath: psychImg,
+        section: 'Psychiatry Section',
+        questions: '50 Question',
+      },
+    ],
+  };
+
+  search=(e) => {
```

Comment:
this function does nothing.
Since we are including dummy data, I think we should implement functionality to allow us to search through it.